build/target-repository/docs/rector_rules_overview.md

```diff
@@ -6506,6 +6506,19 @@ Refactor Spatie enum class to native Enum
 
 ## Php82
 
+### FilesystemIteratorSkipDotsRector
+
+Prior PHP 8.2 FilesystemIterator::SKIP_DOTS was always set and could not be removed, therefore FilesystemIterator::SKIP_DOTS is added in order to keep this behaviour.
+
+- class: [`Rector\Php82\Rector\New_\FilesystemIteratorSkipDotsRector`](../rules/Php82/Rector/New_/FilesystemIteratorSkipDotsRector.php)
+
+```diff
+-new \FilesystemIterator(__DIR__, \FilesystemIterator::KEY_AS_FILENAME);
++new \FilesystemIterator(__DIR__, \FilesystemIterator::KEY_AS_FILENAME | \FilesystemIterator::SKIP_DOTS);
+```
+
+<br>
```

Comment:
documentation needs rebase, you can run:

```
git pull --rebase https://github.com/rectorphp/rector-src.git main
composer docs
git add .
git rebase --continue
git push --force origin filesystem_iterator_skip_dots 
```

Reply:
Hello, I rebased and updted doc